package.json

```diff
@@ -56,6 +56,7 @@
   ],
   "scripts": {
     "build": "node ./lib/mdn-bob.js",
+    "build-pages": "node ./lib/mdn-bob.js --skip-webpack",
```

Comment:
Think we can use a colon instead to follow Yari conventions and the likes?

```suggestion
    "build:pages": "node ./lib/mdn-bob.js --skip-webpack",
```

Reply:
Sure, thanks